src/main/java/org/ow2/proactive/connector/iaas/model/InstanceCredentials.java

```diff
@@ -48,4 +47,9 @@
     private String publicKey;
 
     private String privateKey;
+
+    @Override
+    public String toString() {
```

Comment:
same remark

Reply:
Same here

Reply:
InfrastructureCrendetials > password. PR Updated in that way.